src/core/server/csp/config.ts

```diff
@@ -39,6 +39,13 @@ const getDirectiveValueValidator = ({ allowNone, allowNonce }: DirectiveValidati
 
 const configSchema = schema.object(
   {
+    unsafe_eval: schema.conditional(
+      // Default unsafe_eval to false if it's not a distributable release
+      schema.contextRef('dist'),
+      true,
+      schema.boolean({ defaultValue: true }),
+      schema.boolean({ defaultValue: false })
+    ),
```

Comment:
👏 👏 👏 I like that you have this disabled by default when running from source.

My only nit is that the usage of snake_case for this config setting may be a bit confusing -- the other snake_case settings (`script_src`, `connect_src`, etc.) directly correspond to CSP directives.

WDYT about inverting this setting and changing the name to `disableUnsafeEval`? I feel like that is more consistent with our other optional setting for CSP hardening (`disableEmbedding`), and it more clearly communicates that this setting is not applied by default.

Reply:
Fixed in https://github.com/elastic/kibana/pull/124484/commits/f387fbf5a835defc8cd75c344f4bc46adc53b0ee